DAT_files/hd038087.dat

```diff
@@ -2,10 +2,23 @@
 V = 8.300 +/- 0.010
 B = 8.420 +/- 0.010
 U = 7.970 +/- 0.010
-J = 7.590 +/- 0.020
-H = 7.390 +/- 0.040
-K = 7.250 +/- 0.020
+# 2MASS, IRAC and WISE catalogs all refer to object IC435 when searching for this star. IC435 is a nebula, and not a star. Not sure if photometry should be trusted.
+J = 7.588 +/- 0.024 ; JHK ref = IRSA 2MASS All-Sky Point Source Catalog
+H = 7.386 +/- 0.042
+K = 7.252 +/- 0.023
```

Comment:
IC 435 is the nebula that surrounds this star.  So, I would be suspicious of the photometry too.  Does the plot of the spectra and photometry look ok?

Reply:
The spectrum/SED look good. Probably the automatic identification in the IRSA catalogs are wrongly indicating IC435. I will leave the comment in the file for later reference.